pkg/lint/lintersdb/manager.go

```diff
@@ -272,6 +272,9 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 		linter.NewConfig(golinters.NewNestif()).
 			WithPresets(linter.PresetComplexity).
 			WithURL("https://github.com/nakabonne/nestif"),
+		linter.NewConfig(golinters.NewExportLoopRef()).
+			WithPresets(linter.PresetStyle).
```

Comment:
isn't it `bugs` preset?

Reply:
you are right, bugs present is more appropriate :100: